modules/core/shared/src/main/scala/lucuma/core/model/sequence/StepTime.scala

```diff
@@ -4,41 +4,40 @@
 package lucuma.core.model.sequence
 
 import cats.Eq
+import lucuma.core.util.TimeSpan
 import monocle.Focus
 import monocle.Lens
 import org.typelevel.cats.time._
 
-import java.time.Duration
-
 final case class StepTime(
-  configChange: Duration,
-  exposure:     Duration,
-  readout:      Duration,
-  write:        Duration,
-  total:        Duration
+  configChange: TimeSpan,
+  exposure:     TimeSpan,
+  readout:      TimeSpan,
+  write:        TimeSpan,
+  total:        TimeSpan
```

Comment:
All of these will take at least 0 time or more.

Reply:
Would `total` be calculated from the other times?

Reply:
actually yes, this was extracted from the schema where total is offered but maybe we don't need it here